core-web/libs/dojo-theme/scss/backend/dot-admin/forms/_base.scss

```diff
@@ -333,7 +338,7 @@ dd {
 
     .dijitButton,
     .fakeDojoButton {
-        margin: 0 calc($basic-padding / 2);
+        margin: 0 $basic-padding / 2;
```

Comment:
Check your console on build you should be seeing something like this:

<img width="1247" alt="image" src="https://user-images.githubusercontent.com/751424/214060631-3b371b6d-4396-49bc-a834-9fd1d84c01e0.png">

I updated those when I bring the dojo SCSS to core-web.

Reply:
Noted

Reply:
updated